### PR TITLE
apm/python: move python documentation links to readthedocs

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
### What does this PR do?

Moves all links from being http://pypi.datadoghq.com/trace/docs to https://ddtrace.readthedocs.io/

### Motivation

We're now building versioned docs for the library on readthedocs.

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/compatibility_requirements/python.md [hunk 1]
```diff
@@ -22,60 +22,60 @@ The `ddtrace` library includes support for a number of web frameworks, including
 
 | Framework                 | Supported Version | PyPi Datadog Documentation                                         |
```
Comment (majorgreys) [inline content/en/tracing/compatibility_requirements/python.md:23]:
```suggestion
| Framework                 | Supported Version | Datadog Documentation                                         |
```

File: content/en/tracing/compatibility_requirements/python.md [hunk 2]
```diff
@@ -22,60 +22,60 @@ The `ddtrace` library includes support for a number of web frameworks, including
...
+| [Molten][8]              | >= 0.7.0          | https://ddtrace.readthedocs.io/en/stable/web_integrations.html#molten  |
+| [Pylons][9]              | >= 0.9.6          | https://ddtrace.readthedocs.io/en/stable/web_integrations.html#pylons  |
+| [Pyramid][10]             | >= 1.7            | https://ddtrace.readthedocs.io/en/stable/web_integrations.html#pyramid |
+| [Tornado][11]             | >= 4.0            | https://ddtrace.readthedocs.io/en/stable/web_integrations.html#tornado |
 
 ### Datastore Compatibility
 
 The `ddtrace` library includes support for the following data stores:
 
 | Datastore                          | Supported Version | PyPi Datadog Documentation                                                                    |
```
Comment (majorgreys) [inline content/en/tracing/compatibility_requirements/python.md:40]:
```suggestion
| Datastore                          | Supported Version | Datadog Documentation                                                                    |
```

File: content/en/tracing/compatibility_requirements/python.md [hunk 3]
```diff
@@ -22,60 +22,60 @@ The `ddtrace` library includes support for a number of web frameworks, including
...
+| [Redis][27] [redis-py-cluster][28] | >= 1.3.5          | https://ddtrace.readthedocs.io/en/stable/db_integrations.html#module-ddtrace.contrib.rediscluster |
+| [SQLAlchemy][29]                   | >= 1.0            | https://ddtrace.readthedocs.io/en/stable/db_integrations.html#sqlalchemy                          |
+| [SQLite3][30]                      | Fully Supported   | https://ddtrace.readthedocs.io/en/stable/db_integrations.html#sqlite                              |
+| [Vertica][31]                      | >= 0.6            | https://ddtrace.readthedocs.io/en/stable/db_integrations.html#vertica                             |
 
 ### Library Compatibility
 
 The `ddtrace` library includes support for the following libraries:
 
 | Library           | Supported Version | PyPi Datadog Documentation                                               |
```
Comment (majorgreys) [inline content/en/tracing/compatibility_requirements/python.md:64]:
```suggestion
| Library           | Supported Version | Datadog Documentation                                               |
```

File: content/fr/tracing/compatibility_requirements/python.md [hunk 4]
```diff
@@ -21,60 +21,60 @@ La bibliothèque `ddtrace` prend en charge de nombreux frameworks Web, y compris
 
 | Framework                 | Version prise en charge | Documentation PyPi de Datadog                                         |
```
Comment (majorgreys) [inline content/fr/tracing/compatibility_requirements/python.md:22]:
```suggestion
| Framework                 | Version prise en charge | Documentation de Datadog                                         |
```

File: content/fr/tracing/compatibility_requirements/python.md [hunk 5]
```diff
@@ -21,60 +21,60 @@ La bibliothèque `ddtrace` prend en charge de nombreux frameworks Web, y compris
...
+| [Molten][8]              | >= 0.7.0          | https://ddtrace.readthedocs.io/en/stable/web_integrations.html#molten  |
+| [Pylons][9]              | >= 0.9.6          | https://ddtrace.readthedocs.io/en/stable/web_integrations.html#pylons  |
+| [Pyramid][10]             | >= 1.7            | https://ddtrace.readthedocs.io/en/stable/web_integrations.html#pyramid |
+| [Tornado][11]             | >= 4.0            | https://ddtrace.readthedocs.io/en/stable/web_integrations.html#tornado |
 
 ### Compatibilité des datastores
 
 La bibliothèque `ddtrace` prend en charge les datastores suivants :
 
 | Datastore                          | Version prise en charge | Documentation PyPi de Datadog                                                                    |
```
Comment (majorgreys) [inline content/fr/tracing/compatibility_requirements/python.md:39]:
```suggestion
| Datastore                          | Version prise en charge | Documentation de Datadog                                                                    |
```

File: content/fr/tracing/compatibility_requirements/python.md [hunk 6]
```diff
@@ -21,60 +21,60 @@ La bibliothèque `ddtrace` prend en charge de nombreux frameworks Web, y compris
...
+| [Redis][27] [redis-py-cluster][28] | >= 1.3.5          | https://ddtrace.readthedocs.io/en/stable/db_integrations.html#module-ddtrace.contrib.rediscluster |
+| [SQLAlchemy][29]                   | >= 1.0            | https://ddtrace.readthedocs.io/en/stable/db_integrations.html#sqlalchemy                          |
+| [SQLite3][30]                      | Prise en charge complète   | https://ddtrace.readthedocs.io/en/stable/db_integrations.html#sqlite                              |
+| [Vertica][31]                      | >= 0.6            | https://ddtrace.readthedocs.io/en/stable/db_integrations.html#vertica                             |
 
 ### Compatibilité des bibliothèques
 
 La bibliothèque `ddtrace` prend en charge les bibliothèques suivantes :
 
 | Bibliothèque           | Version prise en charge | Documentation PyPi de Datadog                                               |
```
Comment (majorgreys) [inline content/fr/tracing/compatibility_requirements/python.md:63]:
```suggestion
| Bibliothèque           | Version prise en charge | Documentation de Datadog                                               |
```

File: content/fr/tracing/setup/python.md [hunk 7]
```diff
@@ -103,9 +103,9 @@ tracer.configure(
 [4]: /fr/tracing/setup/docker/
 [5]: /fr/agent/kubernetes/apm/
 [6]: http://pypi.datadoghq.com/trace/docs
```
Comment (majorgreys) [inline content/fr/tracing/setup/python.md:105]:
```suggestion
[6]: https://ddtrace.readthedocs.io/en/stable/
```

File: content/ja/tracing/setup/python.md [hunk 8]
```diff
@@ -103,9 +103,9 @@ tracer.configure(
 [4]: /ja/tracing/setup/docker/
 [5]: /ja/agent/kubernetes/apm/
 [6]: http://pypi.datadoghq.com/trace/docs
```
Comment (majorgreys) [inline content/ja/tracing/setup/python.md:105]:
```suggestion
[6]: https://ddtrace.readthedocs.io/en/stable/
```

**Review by majorgreys — COMMENTED** [review]
A few more replacements for pypi links and references in docs

**Comment by Kyle-Verhoog** [comment]
@majorgreys how do you feel about doing "Library documentation" vs "Datadog documentation". "Datadog documentation" sounds a bit like the corp site docs themselves.

**Review by kayayarai — APPROVED** [review]
lgtm from documentation perspective. I agree that the headings should be "Library documentation" not "Datadog documentation." @Kyle-Verhoog Let me know when you commit that change and I'll merge this.

**Comment by Kyle-Verhoog** [comment]
@kayayarai done! I didn't update it in the fr/jp docs since I think there's a translation process for that. Let me know if this is alright 🙂

Also note that we have #8469 but we can merge these in any order as there will be conflicts either way.

**Comment by kayayarai** [comment]
> I didn't update it in the fr/jp docs since I think there's a translation process for that. Let me know if this is alright 🙂
> 
That is correct. I'll merge this now.

**Pull request merged**